tool/templates/cpp2/base.txt

```diff
@@ -0,0 +1,35 @@
+#ifndef {{ header_guard }}
```

Comment:
nit: probably should use .hpp or .hpp.tmpl

(and set gitattributes to highlight tmpl as Jinja or something)

Reply:
I'll use `.h.jinja` since that extension is picked up by the plugins for VSCode and Sublime Text

https://marketplace.visualstudio.com/items?itemName=samuelcolvin.jinjahtml